### PR TITLE
fix: add issue template

### PR DESCRIPTION
**Pull request by bcullman**
this will pre-populate the textarea seen when creating a new issue.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+### Version
+**fundamental-react:**
+**fiori-fundamentals:**
+
+---
+
+_**NOTE:** Where applicable, please include uncropped screen captures._
+
+_**DISCLAIMER:**
+After triaging an issue, the fundamental-react-team will see if it can be reproduced or confirmed. If more information is needed, the fundamental-react-team team will contact the author.  Any issues awaiting responses from the author for more than 7 days will be closed.  The author can re-open the issue at a later time if they can present the requested information._
```
Comment (mobot11) [inline .github/ISSUE_TEMPLATE.md]: Should probably be fundamental-react team instead of fundamental-react-team.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+### Version
+**fundamental-react:**
+**fiori-fundamentals:**
+
+---
+
+_**NOTE:** Where applicable, please include uncropped screen captures._
+
+_**DISCLAIMER:**
+After triaging an issue, the fundamental-react team will see if it can be reproduced or confirmed. If more information is needed, the fundamental-react team team will contact the author.  Any issues awaiting responses from the author for more than 7 days will be closed.  The author can re-open the issue at a later time if they can present the requested information._
```
Comment (mobot11) [inline .github/ISSUE_TEMPLATE.md]: Looks like there's an extra _team_ after if more _information is needed, the fundamental-react team_ now.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+### Description
+
+
+### Version
```
Comment (greg-a-smith) [inline .github/ISSUE_TEMPLATE.md]: Maybe ...
```markdown
### Versions
```

**Comment by greg-a-smith** [comment]
@bcullman If we haven't already started, maybe a PR template would be good too?  We could add a placeholder in the template for the "Fixes..." notation since it's very easy to forget.

**Comment by bcullman** [comment]
@greg-a-smith good idea. added.

**Pull request merged**